speaker_verification/tests/test_sqlmethods.py

```diff
@@ -0,0 +1,89 @@
+import unittest
+
+import pytest
+import numpy as np
+from os.path import join, abspath, dirname
+import pathlib
+import sqlite3
+
+from speaker_verification.sql_utils import(
+    create_db_table,
+    insert_db_row,
+    select_db_row,
+    remove_db_row
+)
+
+table_id = 219944922
+mfcc=np.array([1,2,3])
+
+def test_database():
+    """Method test for valid datbase"""
+    try:
+        assert select_db_row("asbjasn",111111111)
+    except AssertionError:
+        return "Datbase does not exist"
```

Comment:
Rather than the function returns a constant string in line 24, I think you should return a constant variable which is assigned to a const string; e.g. `DATA_NOT_EXIST_ERROR = "Database does not exist"`. Also, you should be careful with spelling `Database` instead of `Datbase`. In addition, the better solution, you can create a class or a file which contains all database-error-related constant variables.

Reply:
I'll remove the changes made to other files besides denoise.py since its not part of this task